src/components/Header.js

```diff
@@ -0,0 +1,109 @@
+//import {useState} from 'react';
+import styled from "styled-components";
+
+export default function Header({showmessages}) {
+
+return (
+    <Top>
+      <Smile>
+        <Eye>
+          <div></div>
+          <div></div>
+        </Eye>
+        <Mouth />
+      </Smile>
+      <Speechbubble><div></div>
```

Comment:
Are these `div`s necessary? If so and if they have the same styling, I would recommend to add a new styled component with a good and descriptive name so that it is easier to understand what and why they are there.